wandb/sdk/launch/launch.py

```diff
@@ -0,0 +1,199 @@
+    docker_args: Optional[Dict[str, Any]],
+    resource: str,
+    launch_config: Optional[Dict[str, Any]],
+    synchronous: Optional[bool],
+    api: Api,
+) -> AbstractRun:
+    """
+    Helper that delegates to the project-running method corresponding to the passed-in backend.
+    Returns a ``SubmittedRun`` corresponding to the project run.
+    """
+
+    launch_spec = construct_launch_spec(
+        uri,
+        experiment_name,
+        wandb_project,
+        wandb_entity,
+        docker_image,
+        entry_point,
+        version,
+        parameters,
+        launch_config,
+    )
+    project = create_project_from_spec(launch_spec, api)
+    project = fetch_and_validate_project(project, api)
+
+    # construct runner config.
+    runner_config: Dict[str, Any] = {}
+    runner_config[PROJECT_SYNCHRONOUS] = synchronous
+    runner_config[PROJECT_DOCKER_ARGS] = docker_args
+
+    backend = loader.load_backend(resource, api, runner_config)
+    if backend:
+        submitted_run = backend.run(project)
+        return submitted_run
+    else:
+        raise ExecutionException(
+            "Unavailable backend {}, available backends: {}".format(
+                resource, ", ".join(loader.WANDB_RUNNERS.keys())
+            )
+        )
+
+
+def run(
+    uri: str,
+    api: Api,
+    entry_point: Optional[str] = None,
+    version: Optional[str] = None,
+    parameters: Optional[Dict[str, Any]] = None,
+    docker_args: Optional[Dict[str, Any]] = None,
+    experiment_name: Optional[str] = None,
+    resource: str = "local",
+    wandb_project: Optional[str] = None,
+    wandb_entity: Optional[str] = None,
+    docker_image: Optional[str] = None,
+    config: Optional[Dict[str, Any]] = None,
+    synchronous: Optional[bool] = True,
+) -> AbstractRun:
+    """
+    Run a W&B launch experiment. The project can be wandb uri or a Git URI.
+    :raises: :py:class:`wandb.exceptions.ExecutionException` If a run launched in blocking mode
+             is unsuccessful.
+    :param uri: URI of experiment to run. A wandb run uri or a Git repository URI.
+    :param api: An instance of a wandb Api from wandb.apis.internal.
+    :param entry_point: Entry point to run within the project. Defaults to using the entry point used
+                        in the original run for wandb URIs, or main.py for git repository URIs.
+    :param version: For Git-based projects, either a commit hash or a branch name.
+    :param parameters: Parameters (dictionary) for the entry point command. Defaults to using the
+                       the parameters used to run the original run.
+    :param docker_args: Arguments (dictionary) for the docker command.
+    :param experiment_name: Name of experiment under which to launch the run.
+    :param resource: Execution backend for the run: W&B provides built-in support for "local" backend
+    :param wandb_project: Target project to send launched run to
+    :param wandb_entity: Target entity to send launched run to
```

Comment:
Since users will only use this command within the context of `wandb.run` there's no reason to prepend these arguments with `wandb_` that I can think of...

Reply:
We can swap these back over to entity/project. The main reason for prepending with `wandb_` was to differentiate the project class arg from the target project.

I'm going to change the `Project` class to `LaunchProject` referred to as `launch_project` and drop these prepended wandb_